client/src/sendMessageArea.tsx

```diff
@@ -122,11 +122,14 @@ export const SendMessageArea = ({
     confirmClaimable(verifiedGroupMentionShorthands?.length > 0).then(
       (claimable) =>
         (tourProgress.isRunning
-          ? tourProgress.sendItem(() => {
+          ? (sendTelemetryEvent?.(PINBOARD_TELEMETRY_TYPE.INTERACTIVE_TOUR, {
+              tourEvent: "messaging",
+            }),
+            tourProgress.sendItem(() => {
               setMessage("");
               clearPayloadToBeSent();
               setUnverifiedMentions([]);
-            })
+            }))
```

Comment:
this looks a little odd, I'd have thought the telemetry event would be called in that callback which clears the message etc.

Reply:
Seemed like the only way to keep the ternary (to avoid having to repeat lines 135-143 in this file) while calling multiple functions ([reference](https://github.com/guardian/pinboard/pull/254/files/91bff7bb8dd2dfc04859b1c367ddedbddcb6bb8d#diff-a2a48a4733d8d8a4c55d4e1013b725684b24485ca744531d25abf49f4e3a13b6R136-R143)), but maybe I should just use if/else for more clarity.